src/garage/np/algos/cem.py

```diff
@@ -106,16 +102,16 @@ def train_once(self, itr, paths):
         """
         paths = self.process_samples(itr, paths)
 
-        epoch = itr // self.n_samples
-        i_sample = itr - epoch * self.n_samples
+        epoch = itr // self.n_epoch_cycles
```

Comment:
uh is this what you intended in this algo? i think the original name was accurate. also, n_samples is an import hyperparam for CEM

Reply:
Are you suggesting we should bring back it as a parameter for CEM?

Reply:
I am suggesting we should keep n_samples, because it is different from n_epoch_cycles.